components/button/stories/button.stories.js

```diff
@@ -317,6 +317,8 @@ DisableWrapping.tags = ["!dev"];
 DisableWrapping.storyName = "Text overflow - disabled text wrap";
 DisableWrapping.args = {
 	variant: "primary",
+	label: "Be a premium member",
+	noWrap: true,
```

Comment:
These 2 args were being repeated in each template in `TextWrapTemplate`, so I moved them here (it also made it clearer to me to see the `noWrap: true` in the story args as well).